py3status/modules/wifi.py

```diff
@@ -63,16 +64,44 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = 'W: [{bitrate} {signal_percent} {ssid}|\?show down]'
```

Comment:
It works out of the box  as-is. I glare things over, self-review, etc. I was wondering if we should explicitly test `ssid` to make sure we only explicitly test `ssid` or if that's unnecessary and we keep it out because it works without `if=ssid`, but seems like a slightly code change to test them all.

Part of the code changes is that we always try to remove unnecessary crufts so I was wondering if that's the case here or not.

It's similar to testing `deadbeef`... Instead of just testing `isplaying`, I test popular parameter (ie `title`) to help with backward compatible.